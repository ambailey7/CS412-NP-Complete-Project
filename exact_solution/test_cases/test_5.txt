15 105
a b 12
a c 24
a d 21
a e 17
a f 45
a g 12
a h 11
a i 48
a j 36
a k 64
a l 11
a m 7
a n 4
a o 42
b c 29
b d 20
b e 75
b f 35
b g 88
b h 52
b i 57
b j 35
b k 67
b l 40
b m 75
b n 42
b o 96
c d 99
c e 6
c f 28
c g 93
c h 8
c i 70
c j 4
c k 36
c l 4
c m 8
c n 96
c o 43
d e 42
d f 8
d g 37
d h 8
d i 55
d j 18
d k 18
d l 96
d m 68
d n 2
d o 33
e f 74
e g 40
e h 72
e i 58
e j 94
e k 65
e l 1
e m 15
e n 74
e o 90
f g 52
f h 59
f i 15
f j 43
f k 73
f l 83
f m 61
f n 92
f o 44
g h 14
g i 20
g j 2
g k 29
g l 78
g m 83
g n 34
g o 62
h i 33
h j 43
h k 52
h l 84
h m 33
h n 24
h o 5
i j 11
i k 21
i l 62
i m 77
i n 48
i o 89
j k 43
j l 61
j m 37
j n 89
j o 61
k l 28
k m 59
k n 91
k o 98
l m 91
l n 68
l o 48
m n 90
m o 76
n o 74