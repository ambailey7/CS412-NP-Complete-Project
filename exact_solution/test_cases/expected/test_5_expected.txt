1989.452
3 9 8 10 12 0 7 4 11 6 2 5 1 3